doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -252,11 +252,10 @@ Complete the following steps:
 
       nRF Cloud - Add New window
 
-#. Activate your iBasis SIM card by entering the 18-digit :term:`Integrated Circuit Card Identifier (ICCID)` and the :term:`Personal Unblocking Key (PUK)`, accepting the Terms and the Privacy Policy, and clicking :guilabel:`Activate SIM`.
+#. Activate your iBasis SIM card by entering the 18-digit :term:`Integrated Circuit Card Identifier (ICCID)` or the 19-digit :term:`eUICC Identifier (EID)`, and the :term:`Personal Unblocking Key (PUK)`, accepting the Terms and the Privacy Policy, and clicking :guilabel:`Activate SIM`.
 
-   The ICCID and the PUK are printed on the SIM card.
+   The ICCID,EID and the PUK are printed on the SIM card.
```

Comment:
not sure what the comma usage guidelines are... 
`The ICCID, EID and the PUK` -->`The ICCID, EID, and the PUK ` ?